src/librustdoc/html/static/js/src-script.js

```diff
@@ -100,11 +97,16 @@ window.rustdocToggleSrcSidebar = () => {
 
 // This function is called from "src-files.js", generated in `html/render/write_shared.rs`.
 // eslint-disable-next-line no-unused-vars
-function createSrcSidebar() {
+/**
+ * @param {string} srcIndexStr - strinified json map from crate name to dir structure
+ */
+function createSrcSidebar(srcIndexStr) {
+    console.log(srcIndexStr);
```

Comment:
Another one? 😅 https://github.com/rust-lang/rust/pull/142100/files#r2131358481
I think we have eslint run on tidy, and it can catch these `console.log`s... might be an easy and helpful change to enable that lint.

Reply:
I'll add this check. We can handle each remaining `console.log` by telling eslint to ignore them.

Thanks @yotamofek!